parse.go

```diff
@@ -55,9 +55,17 @@ var (
 	ErrorMismatchPixelDataLength = errors.New("the size calculated from DICOM elements and the PixelData element's VL are mismatched")
 )
 
+func Parse(in io.Reader, bytesToRead int64, frameChan chan *frame.Frame, opts ...ParseOption) (Dataset, error) {
+	return parseInternal(in, bytesToRead, frameChan, opts...)
+}
+
+func ParseUntilEOF(in io.Reader, frameChan chan *frame.Frame, opts ...ParseOption) (Dataset, error) {
+	return parseInternal(in, -1, frameChan, opts...)
```

Comment:
I wonder if it might be more readable both here and in reader.go to use a sentinel value to indicate a "limit" that indicates read till EOF. For example in reader.go we could have 

```go
const LimitReadUntilEOF = -9999
```

 and then elsewhere in reader.go instead of 
```go
// if limit < 0 than we should read until EOF
return (r.BytesLeftUntilLimit() <= 0 && r.limit > 0)
```
we can have
```go
// if limit < 0 than we should read until EOF
return (r.BytesLeftUntilLimit() <= 0 && r.limit != LimitReadUntilEOF)
```

Reply:
sounds good, I'll put these changes in place and check that it all works